src/plugins/foxPage/components/MainOpportunity.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, Text as CText } from '@chakra-ui/react'
+import { useTranslate } from 'react-polyglot'
+import { fox } from 'test/mocks/assets'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+import { Card } from 'components/Card/Card'
+import { Text } from 'components/Text/Text'
+import { BigNumber } from 'lib/bignumber/bignumber'
+
+type MainOpportunityProps = {
+  apy: string
+  tvl: BigNumber
+  balance: string
+  onClick: () => void
+}
+
+export const MainOpportunity = ({ apy, tvl, balance, onClick }: MainOpportunityProps) => {
```

Comment:
I'm wondering if `MainOpportunity` is the best naming here. It's very ambiguous when we consider non-fox opportunities down the road.

Perhaps `FoxOpportunity` might be better to clarify that this is directly related to general Fox earning opportunities.